### PR TITLE
add a code example to how-to-create-and-use-shared-ptr-instances.md

### LINKED ISSUE #4461: Example 5
Would be nice if Example 5 had actual code examples.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 69cc6836-826c-9f95-0b59-29bf9180d343
* Version Independent ID: 29f62ae3-670b-9fe6-b938-e1441d4a8e3c
* Content: [How to: Create and use shared_ptr instances](https://learn.microsoft.com/en-us/cpp/cpp/how-to-create-and-use-shared-ptr-instances?view=msvc-170)
* Content Source: [docs/cpp/how-to-create-and-use-shared-ptr-instances.md](https://github.com/MicrosoftDocs/cpp-docs/blob/main/docs/cpp/how-to-create-and-use-shared-ptr-instances.md)
* Product: **visual-cpp**
* Technology: **cpp-language**
* GitHub Login: @TylerMSFT
* Microsoft Alias: **twhitney**

### PR DESCRIPTION
**Pull request by fsb4000**
Fixes #4461

### CODE REVIEW

**Comment by Court72** [comment]
@TylerMSFT

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by TylerMSFT — CHANGES_REQUESTED** [review]
Please update the date of the article since a meaningful change is happening. It helps us with our tools that track freshness.

**Review by TylerMSFT — APPROVED** [review]

**Review by TylerMSFT — APPROVED** [review]

File: docs/cpp/codesnippet/CPP/how-to-create-and-use-shared-ptr-instances_5.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    // Pass the shared_ptr by reference or const reference.
+    // In this case, the reference count isn't incremented.
+    use_shared_ptr_by_reference(sp);
+    use_shared_ptr_by_const_reference(sp);
+
+    // Pass the underlying pointer or a reference to the underlying object.
+    use_raw_pointer(sp.get());
+    use_reference(*sp.get());
+}
```
Comment (CaseyCarter) [inline docs/cpp/codesnippet/CPP/how-to-create-and-use-shared-ptr-instances_5.cpp:24]: Is it worth adding another line at the end with `use_shared_ptr_by_value(std::move(sp))` which doesn't increment the reference count but in fact transfers ownership to the callee?

**Review by TylerMSFT — CHANGES_REQUESTED** [review]
Please makes the updates that Casey suggested, then we should be good to go.

**Review by TylerMSFT — APPROVED** [review]
This is great - thank you @fsb4000

**Comment by TylerMSFT** [comment]
#sign-off

File: docs/cpp/codesnippet/CPP/how-to-create-and-use-shared-ptr-instances_5.cpp (hunk 1 above)
Comment (CaseyCarter) [reply docs/cpp/codesnippet/CPP/how-to-create-and-use-shared-ptr-instances_5.cpp:29]: Thanks, @fsb4000, this looks great.

**Review by CaseyCarter — APPROVED** [review]

**Pull request merged**